tests/phpunit/includes/abstract-testcase.php

```diff
@@ -1070,6 +1070,41 @@ public function assertNonEmptyMultidimensionalArray( $actual, $message = '' ) {
 		}
 	}
 
+	/**
+	 * Assert that two text strings representing file paths are the same, while ignoring
+	 * OS-specific differences in the directory separators.
+	 *
+	 * This allows for tests to be compatible for running on both *nix based as well as Windows OS.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @param string $path_a File or directory path.
+	 * @param string $path_b File or directory path.
+	 */
+	public function assertSamePathIgnoringDirectorySeparators( $path_a, $path_b ) {
+		$path_a = $this->normalizeDirectorySeparatorsInPath( $path_a );
+		$path_b = $this->normalizeDirectorySeparatorsInPath( $path_b );
+
+		$this->assertSame( $path_a, $path_b );
+	}
+
+	/**
+	 * Normalize directory separators in a file path to be a forward slash.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @param string $path File or directory path.
+	 *
+	 * @return string
+	 */
+	public function normalizeDirectorySeparatorsInPath( $path ) {
```

Comment:
As this method is not an expect or assert, shouldn't it use the WordPress name convention?

```suggestion
	public function normalize_directory_separators_in_path( $path ) {
```

Reply:
The use of camelCase is deliberate for consistency with other expectation/assertion methods in the `TestCase` file.

Reply:
But in this case, it's a public helper function and not an expectation or assertion. It is used by a new assertion, yes. But it could also be used within a test class / method as a helper for setting up conditions before doing an assertion or expectation, such as being done in this PR in `test_get_block_templates_paths_dir_exists()`.

As it's a helper, I'm wondering if it should follow snake_case, similar to other helper methods.

Reply:
Well, as I hinted at before, quite a lot of those helpers in the `TestCase` use `camelCase`, so I still think this is most consistent. Some examples:
* `skipOnAutomatedBranches()`
* `expectDeprecated()`
* `setExpectedIncorrectUsage()`
* `knownWPBug()`
* `prepareTemplate()`

Note: AFAICS none of these are required to be camelCase by PHPUnit, still they are.

Reply:
You're right. There are mixed naming conventions for the helpers, i.e. some are camelCase (as you note) and others are snake_case.

I don't have strong opinions. Given precedence, seems okay to use camelCase.

Note: Though out of scope for this PR, might be worth establishing a code standard for test case helper methods and updating the handbook.

Reply:
> Note: Though out of scope for this PR, might be worth establishing a code standard for test case helper methods and updating the handbook.

I vaguely remember we opened a ticket about this quite a while back (including about naming conventions for data providers and such). Would need to search for it, but I think it should still be open.